content/spin/managing-cache.md

```diff
@@ -0,0 +1,77 @@
+title = "Managing Cache"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/spin/managing-cache.md"
+
+---
+
+- [The Spin Cache](#the-spin-cache)
+- [Viewing Cached Files](#viewing-cached-files)
+- [Clearing Cached Files](#clearing-cached-files)
+
+## The Spin Cache
+
+A running Spin application can [fetch resources from remote registries](https://developer.fermyon.com/spin/spin-oci) and individual component sources via HTTP endpoints. These resources naturally consume network bandwidth. To ensure network efficiency, and to prevent waiting to download the same files every time an application is started, Spin automatically maintains a local `spin/registry` cache.
+
+## Viewing Cached Files
+
+All downloadable application files that are automatically cached can be found in the following file paths, depending on the operating system being used to run the Spin application:
+
+| Platform | Value | Example |
+| :--- | :--- | :--- |
+| Linux | `$XDG_CACHE_HOME` or `$HOME/.cache/spin/registry` | `/home/alice/.cache/spin/registry` |
+| macOS | `$HOME/Library/Caches/spin/registry` | `/Users/Alice/Library/Caches/spin/registry` |
+| Windows | `{FOLDERID_LocalAppData}/spin/registry` | `C:\Users\Alice\AppData\Local/spin/registry` |
+
+For example inspecting the local `spin/registry` cache on macOS, (using the `tree` command) would look like this:
+
+<!-- @selectiveCpy -->
+
+```console
+$ tree ~/Library/Caches/spin/registry/
+
+├── data
```

Comment:
This is a phenomenal amount of detail to go into, and maybe a bit overwhelming?  I am not sure we would even guarantee the stability of this layout.  At this level of depth it feels more like "debugging Spin internals" kind of info rather than what an application developer would need to know.

Reply:
Yes, this is a great point. The amount of page real estate this takes up is a bit much. I will trim it down but keep the layout because this is all by design. I think we need to find the middle ground between verbose and black box, right? :)

Let me trim it and we can see what it looks like.

Reply:
Please check the latest commit for updates.